config/runtime.exs

```diff
@@ -136,13 +136,6 @@ if config_env() == :dev do
         team_id: System.fetch_env!("APNS_TEAM_ID"),
         topic: System.fetch_env!("APNS_TOPIC"),
         env: :dev
-      },
-      %{
-        key: System.fetch_env!("PROD_APNS_KEY"),
-        key_id: System.fetch_env!("PROD_APNS_KEY_ID"),
-        team_id: System.fetch_env!("APNS_TEAM_ID"),
-        topic: System.fetch_env!("APNS_TOPIC"),
-        env: :prod
```

Comment:
deleted to not accidentally send pushes to prod while in dev
@ruslandoga, or are pushes to prod disabled in some other place as well?

Reply:
no, you are right, prod pushes can be sent from dev env